conf/cityparams.conf

```diff
@@ -90,6 +90,28 @@ city-params {
     burnaby = "private"
     teaneck-nj = "public"
   }
+  launch-date {
+    newberg-or = "2019-01-31"
+    washington-dc = "2015-10-17"
+    seattle-wa = "2019-03-12"
+    columbus-oh = "2019-11-12"
+    cdmx = "2020-01-09"
+    spgg = "2020-05-30"
+    pittsburgh-pa = "2020-07-30"
+    chicago-il = "2021-09-03"
+    amsterdam = "2022-03-17"
+    la-piedad = "2022-03-17"
+    oradell-nj = "2022-03-17"
+    validation-study = "2022-05-10"
+    zurich = "2022-11-08"
+    taipei = "2023-01-24"
+    new-taipei-tw = "2023-08-06"
+    keelung-tw = "2023-08-10"
+    auckland = "2023-02-14"
+    cuenca = "2023-04-02"
+    burnaby = "2023-08-09"
+    teaneck-nj = "2023-08-24"
```

Comment:
You should pull in most recent changes from develop, which will reveal that we have a new deployment in Walla Walla, WA! For now let's use "2023-10-05" for the launch date for that server.